jupyterhub/templates/image-puller/serviceaccount.yaml

```diff
@@ -2,18 +2,16 @@
 ServiceAccount for the pre-puller hook's image-awaiter-job
 */}}
 {{- if .Values.prePuller.hook.serviceAccount.create -}}
-{{- if (include "jupyterhub.imagePuller.daemonset.hook.install" .) -}}
+{{- if or (include "jupyterhub.imagePuller.daemonset.hook.install" .) .Values.prePuller.continuous.enabled -}}
```

Comment:
There are two types of image-pullers (daemonset-continuous and daemonset-hook). If either of them is enabled, we need to create the service account.